### PR TITLE
mikes branch

### PR DESCRIPTION
**Pull request by gottamike**
removed "foo bar" added my name

### CODE REVIEW

**Comment by gottamike** [comment]
Sierra may or may not have helped me with this branch...

File: students/mike.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Methods 2022 - Students</title>
```
Comment (sleddog) [inline students/mike.html:7]: add your name to the title
```
<title>Methods 2022 - Students - Mike Gotta</title>
```

File: students/mike.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Methods 2022 - Students- Mike Gotta</title>
```
Comment (Sawahh) [inline students/mike.html:7]: fix spacing

File: students/mike.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+    <h1>Testing</h1>
```
Comment (Sawahh) [inline students/mike.html:10]: Testing what?

File: students/mike.html (hunk 1 above)
Comment (gottamike) [reply students/mike.html]: I did this!

**Review by sleddog — APPROVED** [review]

**Pull request merged**